test/powershell/Common/TestHostCS.psm1

```diff
@@ -290,7 +304,7 @@ function New-TestHost
     }
 
     if ( ! ("TestHost.TestHost" -as "type" )) {
-       $t = add-Type -pass $definition -ref $references
+       $t = add-Type -pass $definition -ref $references -WarningAction SilentlyContinue 

```

Comment:
I check CI logs - no warnings. I see the warnings only locally. 
So revert the change.